entries/h/hirezstudios.com.json

```diff
@@ -1,8 +1,6 @@
 {
   "Hi-Rez Studios": {
     "domain": "hirezstudios.com",
-    "url": "https://www.hirezstudios.com/",
```

Comment:
No response on https://hirezstudios.com/. I suggest we leave the url tag

Reply:
It worked for me both in Chrome and Firefox (and redirected to www.hirezstudios.com), but wget and curl gave no reponse. Sigh...

Reply:
Your browsers probably tried to connect to the HTTP version when HTTPS failed. I've disabled that in about:config 😬